modules/nf-core/calder2/main.nf

```diff
@@ -0,0 +1,43 @@
+def VERSION = '0.3'
```

Comment:
This needs to be in the script block: https://github.com/nf-core/modules/blob/f919110f52861a5c0c2ab8229c17560f41405721/modules/nf-core/homer/annotatepeaks/main.nf#L26